components/Forms/Select/select.css

```diff
@@ -47,6 +47,11 @@
   @apply tw-flex tw-align-middle tw-justify-between;
 }
 
+.fui-select-container .fui-select__section-header {
+  font-size: 15px;
```

Comment:
Is there a way to do with tailwind `@apply` rules?

Reply:
Probably! Have never used tailwind, will try to look for the correct way to do this.